prerelease_website/static/css/index.css

```diff
@@ -51,20 +51,6 @@ html, body {
   color: #46B6EF;
 }
 
-.jade-column
-{
-  height: 100%;
-  background: #19A36A;
-  width: 25%;
```

Comment:
Do we need to resize the other column to 33% if we go back to only 3 columns ?
(I didnt try to deploy the change so maybe it's not necessary)

Reply:
Not sure, I didn't try it either. I'll check locally and see.

Reply:
I didn't see an issue with the 25%, but it also doesn't make sense and looks ok with 33%, so I changed it.